xm-commons-domain-event/src/main/resources/config/liquibase/changelog/outbox/00000000000000_initial_schema.yaml

```diff
@@ -0,0 +1,82 @@
+databaseChangeLog:
```

Comment:
in xm we using xml for liquibase changelog, why here yml?

Reply:
@sergeysenja1992 this is mandatory requirement? If yes we will change this.

xml old unclear format, yml modern that xml.